lib/request.coffee

```diff
@@ -172,6 +174,8 @@ module.exports = getRequest = ({
 	# .get('body')
 	###
 	exports.send = (options = {}) ->
+		{ responseFormat } = options
+		delete options.responseFormat
```

Comment:
We actually don't need to delete this custom prop as we fully own the options and whitelist them before passing down to `fetch`